bot/telegram_bot.py

```diff
@@ -198,11 +285,25 @@ async def image(self, update: Update, context: ContextTypes.DEFAULT_TYPE):
         image_query = message_text(update.message)
         if image_query == '':
             await update.effective_message.reply_text(
+
                 message_thread_id=get_thread_id(update),
                 text=localized_text('image_no_prompt', self.config['bot_language'])
             )
             return
 
+        """
+        Store this bot command id
+        """
+        if update.message.text.startswith("/image"):
+
+            if chat_id not in self.bot_commands:
```

Comment:
`chat_id` is not in the scope here. You'll need to declare it somewhere before that, i.e:
```
chat_id = update.effective_chat.id
```